client/src/components/profileInfoEdit/ProfileInfoEdit.css

```diff
@@ -33,4 +19,45 @@
 
 .placeholder-text::placeholder {
   color: #A9A9A9
+}
+
+.profile-img {
+  display: block;
+  margin: 20px auto;
+  object-fit: cover;
+}
+
+.profile-pic {
+  width:0;
```

Comment:
nit: not sure if it is necessary to add space after `:`  in CSS file. 🤔

Reply:
I guess we should, there isn't an eslint for css yet... so I might've missed that 😆